content/blog/hello-world.md

```diff
@@ -1,28 +1,40 @@
 
-## What this is about
+## What are we doing?
 
-{{< rawhtml >}}<span class="backticked">scverse</span>{{< /rawhtml >}} is a consortium for joint maintenance of core packages in omics data analysis in life sciences. It has been founded to ensure the long-term maintenance of the core infrastructure in the field.
+Our first priority is to build a strong and lasting community around the tools and the associated analysis. The knowledge to build and maintain our tools should be shared among many to avoid single points of failures. Therefore, we are building tooling to more easily generate scverse based packages and analyses to grow the ecosystem. Furthermore, to bring the community together we are planning workshops and hackathons which will be open to the general community. We are also hoping that a healthy scverse community will naturally guide us to develop tooling where it is the most required.
 
+There are already a few visible outcomes of our joint work including improved 
+[mudata performance](https://mudata.readthedocs.io/en/latest/changelog.html#v0-1-2) and
+[mudata documentation](https://mudata.readthedocs.io/en/latest/). Additionally, 
+[scvi-tools is now using MuData](https://github.com/scverse/scvi-tools/pull/1444) for the representation of multimodal data. 
+[Scanpy has gained new workshop notebooks](https://github.com/scverse/scanpy-tutorials/pull/52), and, moreover, we have started 
+[a new page with tutorials for scverse libraries](https://scverse.org/tutorials/).
 
-## What we are busy with
+## Who is a part of scverse?
 
-While we're continuing to work on the scverse libraries, we're trying to improve how they work and are tested together.
+scverse is explicitly an effort beyond just a single research group. Only together can we continue to ensure that the ecosystem grows and stays interoperable.
+We therefore define several [roles](https://scverse.org/people/) of which only a subset are denoted below.
 
-There are already a few visible outcomes of our joint work including improved {{< rawhtml >}}<a href="https://mudata.readthedocs.io/en/latest/changelog.html#v0-1-2" target="_blank">mudata performance</a>{{< /rawhtml >}} and {{< rawhtml >}}<a href="https://mudata.readthedocs.io/en/latest/" target="_blank">mudata documentation</a>{{< /rawhtml >}}. Things like rich representation for AnnData objects and MuData integration in scvi-tools are work in progress.
+[Isaac Virshup](https://github.com/ivirshup), [Danila Bredikhin](https://github.com/gtca) and [Lukas Heumos](https://github.com/Zethson) form the steering committee and also belong to the core developers together with [Adam Gayoso](https://github.com/adamgayoso), [Giovanni Palla](https://github.com/giovp), [Gregor Sturm](https://github.com/grst), and [Ilia Kats](https://github.com/ilia-kats). The core developers have made significant contributions to the ecosystem and are jointly developing and maintaining the major libraries.
```

Comment:
```suggestion
[Isaac Virshup](https://github.com/ivirshup), [Danila Bredikhin](https://github.com/gtca) and [Lukas Heumos](https://github.com/Zethson) form the steering committee and also belong to the core team along with [Adam Gayoso](https://github.com/adamgayoso), [Giovanni Palla](https://github.com/giovp), [Gregor Sturm](https://github.com/grst), and [Ilia Kats](https://github.com/ilia-kats).
The core developers have made significant contributions to the ecosystem and are jointly developing and maintaining the major libraries.
While all members of the core team are currently core developers, we envision adding a core community role to the project in the near future.
```

Reply:
Thought adding a mention of community team could also be a good place to mention diversity efforts?

Reply:
Is that bad?